extensions/adobe/experience/decisioning/proposition-attribution.example.1.json

```diff
@@ -0,0 +1,12 @@
+{
+  "https://ns.adobe.com/experience/decisioning/attribution": [
+    {
+      "xdm:decisionEventID": "TNT:activity1:experience1:1597208433",
+      "xdm:decisionScope": "__view__"
+    },
+    {
+      "xdm:decisionEventID": "3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
+      "xdm:decisionScope": "OD:eyJhY3Rpdml0eUlkIjoieGNvcmU6b2ZmZXItYWN0aXZpdHk6MTFjZmIxZmE5MzM4MWFjYSIsInBsYWNlbWVudElkIjoieGNvcmU6b2ZmZXItcGxhY2VtZW50OjExNzUwMDk2MTJiMDEwMGMifQ=="
```

Comment:
What's the rationale for this to be base64 encoded? What's the inteneded downstream use?

```
{"activityId":"xcore:offer-activity:11cfb1fa93381aca","placementId":"xcore:offer-placement:1175009612b0100c"}
```

Reply:
This is absolutely **not** a requirement mandated by the schema that is proposed here. From a schema perspective it is string. It is not even an identifier. The semantic the mixin only says to use the string called "scope" that was received with the decision. It could have been a string containing a placement identifier.

Reply:
I changed the example to avoid internal details such as base64 encoded JSON strings for one implementation. The public documentation should not manifest those temporary details